pkgs/development/tools/sauce-connect/default.nix

```
@@ -4,18 +4,18 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "sauce-connect-${version}";
-  version = "4.5.1";
+  version = "4.5.4";
 
   src = fetchurl (
     if stdenv.hostPlatform.system == "x86_64-linux" then {
```

you may want to do an assertion that checks to see if the else branch will land on a darwin system.

```suggestion
    assertOneOf "hostPlatform" stdenv.hostPlatform.system [ "x86_64-linux" "i686-linux" "x86_64-darwin" ]
    if stdenv.hostPlatform.system == "x86_64-linux" then {
```